lib/ansible/modules/cloud/amazon/ec2_vpc_nacl.py

```diff
@@ -369,45 +369,52 @@ def remove_network_acl(client, module):
 #Boto3 client methods
 def create_network_acl(vpc_id, client, module):
     try:
-        nacl = client.create_network_acl(VpcId=vpc_id)
+        if module.check_mode:
+            nacl = dict(NetworkAcl=dict(NetworkAclId="nacl-00000000"))
+        else:
+            nacl = client.create_network_acl(VpcId=vpc_id)
     except botocore.exceptions.ClientError as e:
         module.fail_json(msg=str(e))
     return nacl
 
 
 def create_network_acl_entry(params, client, module):
     try:
-        result = client.create_network_acl_entry(**params)
+        if not module.check_mode:
+            client.create_network_acl_entry(**params)
     except botocore.exceptions.ClientError as e:
         module.fail_json(msg=str(e))
-    return result
```

Comment:
Ah, at first glance I thought this was using [boto2](http://boto.cloudhackers.com/en/latest/ref/vpc.html#boto.vpc.VPCConnection.create_network_acl_entry) which returns a boolean. In the boto3 docs I see it now returns none.